ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -114,8 +114,11 @@
 OVS_FP_BRIDGE_TEMPLATE = 'br-%s-%d'
 OVS_FP_TAP_TEMPLATE = '%s-t%d'
 OVS_BP_TAP_TEMPLATE = '%s-back'
-INJECTED_INTERFACES_TEMPLATE = 'inje-%s-%d'
-MUXY_INTERFACES_TEMPLATE = 'muxy-%s-%d'
+# The name of interface name must be less than 15 bytes.
+# As a result, interface name like 'inje-vms21-6-132' will be invalid and rejected by kernel.
+# Therefore, the name template is shorten to 3 bytes to contain more digits (3 in max)
+INJECTED_INTERFACES_TEMPLATE = 'inj-%s-%d'
+MUXY_INTERFACES_TEMPLATE = 'mux-%s-%d'
```

Comment:
This is a disruptive change. This change will cause interface name change, so testbed deployed before this change cannot be removed after.
How do we deal with that?

Reply:
Thanks for the good catch. I also noticed that some other components that depend on the interface name or bridge name will fail.